### PR TITLE
[stable/sysdig] Fixes compatibility with 1.16

### PR DESCRIPTION
**Pull request by daixiang0**
To support 1.16. A quick overview of deprecated APIs can be found on their blog post: https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/.

Signed-off-by: Xiang Dai 764524258@qq.com

### CODE REVIEW

**Comment by zanhsieh** [comment]
@daixiang0 
Please resolve conflict on stable/sysdig/Chart.yaml. Thank you.

File: stable/sysdig/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: sysdig
-version: 1.4.16
+version: 1.4.17
```
Comment (zanhsieh) [inline stable/sysdig/Chart.yaml]: Please bump to 1.4.18. Thank you.

**Review by zanhsieh — CHANGES_REQUESTED** [review]

**Review by zanhsieh — APPROVED** [review]

**Comment by zanhsieh** [comment]
@nestorsalceda 
Please help.

**Comment by zanhsieh** [comment]
@daixiang0 
Probably another version bump please.

**Review by zanhsieh — APPROVED** [review]

**Review by zanhsieh — APPROVED** [review]

**Comment by bencer** [comment]
Thanks for the PR, can we add a changelog entry?

**Comment by daixiang0** [comment]
@bencer done.

**Pull request merged**